server/sonar-server/src/main/java/org/sonar/server/ws/WsUtils.java

```diff
@@ -109,4 +109,12 @@ public static void checkRequest(boolean expression, List<String> messages) {
 
     return value.get();
   }
+
+  public static <T> T checkStateWithOptional(java.util.Optional<T> value, String message, Object... messageArguments) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Specify a "Object" parameter instead. ([squid:S3553](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS3553))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2FMMF-647%2Fadministration-console-for-webhooks&issues=AWGfGEcdjNA6aRBI4UCQ&open=AWGfGEcdjNA6aRBI4UCQ)

[ISSUE_KEY]: AWGfGEcdjNA6aRBI4UCQ